test/Interop/Cxx/class/type-classification-loadable-silgen.swift

```diff
@@ -54,17 +54,17 @@ func pass(s: StructWithSubobjectDefaultedCopyConstructor) {
 
 // CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithPrivateDefaultedCopyConstructor)
 func pass(s: StructWithPrivateDefaultedCopyConstructor) {
-  // CHECK: bb0(%0 : $*StructWithPrivateDefaultedCopyConstructor):
+  // CHECK: bb0(%0 : $StructWithPrivateDefaultedCopyConstructor):
 }
 
 // CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithInheritedPrivateDefaultedCopyConstructor)
 func pass(s: StructWithInheritedPrivateDefaultedCopyConstructor) {
-  // CHECK: bb0(%0 : $*StructWithInheritedPrivateDefaultedCopyConstructor):
+  // CHECK: bb0(%0 : $StructWithInheritedPrivateDefaultedCopyConstructor):
```

Comment:
There's code in the importer that should make a type address-only if there is any private constructor, do you know why that is not kicking in?